Assets/Scripts/Dialogue/DialogueBubble.cs

```diff
@@ -91,26 +84,135 @@ private IEnumerator DisplayText()
+
+            //If true, we are getting a command.
+            if (currentChar == "<") 
+            {
+                // Go ahead and get the command.
+                while (currentChar != ">" && i < text.Length) 
+                {
+                    currentChar = text[i].ToString();
+                    command += currentChar;
+                    text = text.Remove(i, 1);  // Remove current character. We want to get the next character in the command
+                }
+
+                // Done getting the command
+                if (currentChar == ">") 
+                {
+                    command = command.Trim(tags);
+
+                    if(command.Contains("/"))
+                    {
+                        if(listCommand.Count>0)
+                        {
+                            listCommand[listCommand.Count - 1].EndIndex = i;
+                        }
+                    }
+                    else
+                    {
+                        SpecialCommand newCommand = new SpecialCommand(command, i);
+                        listCommand.Add(newCommand);
+                    }
+
+                    command = "";
+
+                    // Take a step back otherwise a character will be skipped
+                    i--;
+                } 
+                else 
+                {
+                    Debug.Log("Command in dialogue line not closed.");
+                }
+            }
+        }
+
+        return listCommand;
+    }
+
+    private void ExecuteCommand(SpecialCommand command)
+    {
+        if (command == null) 
+        {
+            return;
+        }
+
+        if (command.Name == "shaky") 
+        {
+            StartCoroutine(ShakeTextAt(command.StartIndex, command.EndIndex));
+        }
+    }
+
+    // Check all commands in a given index. 
+    // It's possible to have two commands next to each other in the dialogue line.
+    // This means both will share the same index.
+    private void CheckForCommands(int index) {
+        for (int i = 0; i < SpecialCommands.Count; i++) {
+            if (SpecialCommands[i].StartIndex == index) {
+                ExecuteCommand(SpecialCommands[i]);
+
+                // Remove it
+                SpecialCommands.RemoveAt(i);
+
+                //Take a step back since we removed one command from the list. Otherwise, the script will skip one command.
+                i--;
+            }
+        }
+    }
+
+    // We use regex to strip all <tags> from our current dialogue line
+    // We have two strings: one with tags and the one printing on screen
+    // We keep track of both in order to know when there's a tag to execute, if any
+    private string StripAllTags(string text) 
+    {
+        // Clean string to return
+        string cleanString;
+
+        // Regex Pattern. Remove all "<tag>" from our dialogue line
+        string pattern = "<[^>]+>";
+
+        cleanString = Regex.Replace(text, pattern, "");
```

Comment:
This method can be condensed to 
```
string pattern = "<[^>]+>";

return Regex.Replace(text, pattern, "");
```

No need to extract the `cleanString` local variable since only one operation is performed on it.